packages/svelte-vscode/src/sveltekit/utils.ts

```diff
@@ -26,39 +26,47 @@ export async function findFile(searchPath: string, fileName: string) {
     }
 }
 
-export async function checkProjectType(path: string): Promise<ProjectType> {
+export async function checkProjectKind(path: string): Promise<GenerateConfig['kind']> {
     const tsconfig = await findFile(path, 'tsconfig.json');
     const jsconfig = await findFile(path, 'jsconfig.json');
-    const svelteVersion = await getSvelteVersionFromPackageJson();
-    const isSv5Plus = isSvelte5Plus(svelteVersion);
-    const isTs = !!tsconfig && (!jsconfig || tsconfig.length >= jsconfig.length);
-    if (isTs) {
+
+    const svelteVersion = await getVersionFromPackageJson('svelte');
+    const withRunes = svelteVersion ? versionAtLeast(svelteVersion, 5) : true;
```

Comment:
If for some reason we don't find svelteVersion, fallback to rune mode.
What do you think ?